azure/src/wizard/UserAssignedIdentityCreateStep.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { ManagedServiceIdentityClient } from '@azure/arm-msi';
```

Comment:
Unrelated to the PR, but we should lazy load this Azure package

Reply:
I thought we had linter rule for this... 🤔

Reply:
I think we do in most of the extensions, but probably haven't updated it for the shared packages